docs/contributing.rst

```diff
@@ -148,3 +148,12 @@ Preparing Pull Requests
 
    The merged pull request will undergo the same testing that your local branch
    had to pass when pushing.
+
+#. After your pull request is merged into the `GliderTools/master`, you will need
```

Comment:
I think it might be helpful to explicitly say that this should also be done right **before** someone is planning changes.